docs/current/guides/snippets/ci/nodejs/azure-pipelines.yml

```diff
@@ -0,0 +1,18 @@
+trigger:
+- master
+
+pool:
+  name: 'Default'
+  vmImage: ubuntu-latest
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '18.x'
+  displayName: 'Install Node.js'
+
+- script: npm install @dagger.io/dagger
+  displayName: 'Install Dagger Node.js SDK'
```

Comment:
I noticed that in the other samples in this guide, we are omitting the SDK installation step for all languages. 

https://docs.dagger.io/145912/ci#github-actions
https://docs.dagger.io/145912/ci#circleci
etc.

@jpadams @kpenfound any specific reason for this? Should I also omit it from these samples?

Reply:
Yes, the installation should be from the project's dependencies in Git, not the SDK package manually. I've explained further in the https://github.com/dagger/dagger/pull/5166/commits/14fdcba20cce65466906a634b1295a7cfeb4d058 commit:

> While these snippets were copied from the Azure Pipelines and AWS CodePipeline guides, I favored making them more consistent with the other CI snippets. We can follow up to fix those guides to be more in line with these, but it requires more changes in the text to do that now.
>
> In general terms, the guides instruct how to setup a local environment, try Dagger locally and try it on CI. But in the cookbook, the snippets assume you already have the needed dependencies commited in git (i.e., next phase), so you install the project's dependencies from the directory, not the packages manually (e.g., `... install dagger-io`).